### PR TITLE
Remove implementation guide from distribution

### PR DESCRIPTION
**Pull request by scottcwilson**

### CODE REVIEW

File: docs/INSTALL.TXT [hunk 1]
```diff
@@ -5,19 +5,20 @@ Welcome to Zen Cart(R) E-Commerce
 THE FOLLOWING IS A VERY SIMPLIFIED INSTALLATION DOCUMENT.
 You can find online documentation at: https://docs.zen-cart.com
 
-WE SUGGEST YOU USE THE PDF "Implementation Guide" PDF FILE IN THE /docs FOLDER FOR FULLY DETAILED INSTRUCTIONS.
+WE SUGGEST YOU USE ONLINE DOCS AND THE PDF "Implementation Guide" 
+
+https://docs.zen-cart.com/release/implementation-guide-v157.pdf
```
Comment (brittainmark) [inline docs/INSTALL.TXT:10]: Should this be implementation-guide-v158.pdf.
I know it does not exist but may be confusing if implementing 1.5.8 so would need creating and minor mods

File: docs/INSTALL.TXT [hunk 2]
```diff
@@ -5,19 +5,20 @@ Welcome to Zen Cart(R) E-Commerce
...
+
+FOR FULLY DETAILED INSTRUCTIONS.
 
 
 Zen Cart has a built-in automatic installation system. However to use this you must first unpack and upload the code to a compatible web server:
 
 1. Extract the Zen Cart ZIP file (and tell the unzip utility to *retain* folder structures)
 2. Upload the Zen Cart files and folders to your server using your FTP program
-3. You must make some folders writable. The details are in the /docs/Implementation-Guide.pdf and the HTML-formatted documentation in the /docs/ folder
+3. You must make some folders writable. The details are in the Implementation-Guide.pdf and the online documentation.
```
Comment (brittainmark) [inline docs/INSTALL.TXT:19]: Here you mention Implementation-Guide.pdf here but above called it implementation-guide-v157.pdf
should you include a link? Or point them to Docs>release Documentation>Implementation Guide (PDF)

File: docs/INSTALL.TXT (hunk 1 above)
Comment (scottcwilson) [reply docs/INSTALL.TXT:10]: Thanks. I think we'll just leave off the version number moving forward.

File: docs/INSTALL.TXT (hunk 2 above)
Comment (scottcwilson) [reply docs/INSTALL.TXT:19]: Fixed thanks.

File: docs/INSTALL.TXT (hunk 1 above)
Comment (brittainmark) [reply docs/INSTALL.TXT:10]: Thats fine. The PDF mentions 1.5.7 so will also need changing if you are going to leave it off.

**Comment by scottcwilson** [comment]
I have made a hash out of this. :( 
Starting again.

**Comment by brittainmark** [comment]
Don't feel bad, I do it all the time : )

**Pull request closed without merging**